lib/wallet/bench/latency-bench.hs

```diff
@@ -506,7 +507,7 @@ withShelleyServer tracers action = do
         }
 
     onClusterStart act db (RunningNode conn block0 (np, vData) _) = do
-        listen <- walletListenFromEnv
+        listen <- walletListenFromEnv envFromText
```

Comment:
```suggestion
        let listen = ListenOnRandomPort
```

Turns out that this is perfectly fine for the benchmark.

Reply:
make sense